.github/workflows/ci.yml

```diff
@@ -90,14 +90,11 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      # The beta ARM64 runners don't yet ship with the normal installed tools.
-      - name: Install Docker, Rust and missing development libs (ARM64 only)
+      - name: Install Rust and missing development libs (ARM64 only)
```

Comment:
@dzuelke All of the rest of these install steps aren't needed either. I removed them and my tests pass here:
https://github.com/heroku/buildpacks-python/pull/204

Reply:
I suspected as much, but wanted to verify first :) What about Rust though, do we not want our own, latest, version?

Reply:
There's already a `Update Rust toolchain` step later on for all arches :-)

Reply:
#853